config.py

```diff
@@ -21,6 +21,36 @@
 CODE_URL = 'https://github.com/mne-tools/mne-study-template'
 
 
+# Retrieve custom configuration options
+# -------------------------------------
+#
+# For testing a specific dataset, create a Python file with a name of your
+# liking (e.g., ``mydataset-template-config.py```), and set an environment
+# variable ``MNE_BIDS_STUDY_CONFIG`` to that file.
+#
+# Example
+# ~~~~~~~
+# ``export MNE_BIDS_STUDY_CONFIG=/data/mystudy/mydataset-template-config.py``
+
+if "MNE_BIDS_STUDY_CONFIG" in os.environ:
+    cfg_path = os.environ['MNE_BIDS_STUDY_CONFIG']
+
+    if os.path.exists(cfg_path):
+        print('Using custom configuration specified in MNE_BIDS_STUDY_CONFIG.')
+    else:
+        msg = ('The custom configuration file specified in the '
+               'MNE_BIDS_STUDY_CONFIG environment variable could not be '
+               'found: {cfg_path}'.format(cfg_path=cfg_path))
+        raise ValueError(msg)
+
+    # Import configuration from an arbitrary path without having to fiddle
+    # with `sys.path`.
+    spec = importlib.util.spec_from_file_location(name='custom_config',
+                                                  location=cfg_path)
+    custom_cfg = importlib.util.module_from_spec(spec)
```

Comment:
Well this stuff really shouldn't be here, must have done something wrong while rebasing…